hu.irl.cameractrls.yml

```diff
@@ -0,0 +1,49 @@
+app-id: hu.irl.cameractrls
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+finish-args:
+  - --device=all
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+command: cameractrlsgtk.py
+cleanup:
+  - /include
+  - /lib/cmake
+  - /lib/pkgconfig
+  - '*.a'
+  - '*.la'
+modules:
+  - name: pygobject
+    buildsystem: meson
+    build-options:
+      config-opts:
+        - -Dpycairo=disabled
+        - -Dtests=false
+    sources:
+      - type: archive
+        url: https://gitlab.gnome.org/GNOME/pygobject/-/archive/3.42.1/pygobject-3.42.1.tar.gz
+        sha256: 53fff1eef7f94533f18164c2df2fcc16cd447aac71f34efc8a735a21bc3bebd5
+
+  - name: cameractrls
+    buildsystem: simple
+
+    build-commands:
+      - install -D -t /app/bin cameractrls.py cameractrlsgtk.py cameraview.py
+      - install -D -t /app/bin/images images/icon_256.png
+      - install -D -t /app/share/metainfo hu.irl.cameractrls.metainfo.xml
+      - install -D -T images/icon_256.png /app/share/icons/hicolor/256x256/apps/hu.irl.cameractrls.png
+      - desktop-file-edit
+        --set-icon=hu.irl.cameractrls
+        --set-key=Exec --set-value=cameractrlsgtk.py
+        --remove-key=Path cameractrls.desktop
+      - install -D -T cameractrls.desktop /app/share/applications/hu.irl.cameractrls.desktop
+    post-install:
+      - python3 -m compileall /app/bin
+    sources:
+    - type: file
+      path: hu.irl.cameractrls.metainfo.xml
+    - type: archive
```

Comment:
usually the main source is first (order only matters to humans)

Reply:
this make no sense.

Reply:
The app ID format follows the same convention: org.freedesktop.Platform -> org is the bigger, then freedesktop then the Platform.

Reply:
Why do we want to use two conventions in one file? I prefer the consistency. I think the package related stuff should came first, because it's the natural way if you describe something. (you don't start with the bearings if you describe a car)

Reply:
no, because in an ideal world this file would be in the upstream package so there would be ONE source entry, the one I'm asking be put first. 

Did you submit the metainfo.xml for inclusion in the upstream source (for a future release)?